src/components/DisplayContribution.tsx

```diff
@@ -10,7 +10,7 @@ import { profileScreenStyles } from '../screens/ProfileScreen/styles';
 
 const DisplayContribution = ({ tasks }) => {
   return (
-    <View style={{ padding: 5, width: "100%", alignItems: 'center' }}>
+    <View style={{ padding: 5, width: '100%', alignItems: 'center' }}>
```

Comment:
why single quote change?

Reply:
Happend because of prettier, Ill disable.